packages/fluence-js/src/internal/FluencePeer.ts

```diff
@@ -331,12 +322,8 @@ export class FluencePeer {
                     new Error("Can't use avm: peer is not initialized");
                 }
 
-                const args = JSON.stringify([air]);
-                const rawRes = await this._fluenceAppService!.callService('avm', 'ast', args, undefined);
-                let res;
+                const res = (await this._fluenceAppService!.callService('avm', 'ast', [air], undefined)) as string;
```

Comment:
The same typecasting can also be acoomplished a bit cleaner with
```typescript
const res: string = await this._fluenceAppService!.callService('avm', 'ast', [air], undefined);
```

Reply:
```
Type 'unknown' is not assignable to type 'string'.ts(2322)
```

Reply:
Sorry, my bad, had some problem with VSCode, you are right.
I just hate typecasts in general =)